FoxHound.Web/Program.cs

```diff
@@ -34,23 +34,29 @@ public static void Main(string[] args)
             Host.CreateDefaultBuilder(args)
                 .ConfigureWebHostDefaults(webBuilder =>
                 {
+                    webBuilder.ConfigureAppConfiguration(config => config.AddLocalAppSettings());
```

Comment:
This is essentially what I was trying to do, but it wasn't working for me. Anyways, thanks for figuring this out.